how-to-contribute.md

```diff
@@ -104,74 +99,101 @@ If you want to add a feature, you should bring it up for discussion on the dev@s
 
 In short, communication is a vital part of making a contribution to an Apache project.
 
+<a name="HowtoContribute-git"></a>
+<a name="HowtoContribute-GitUsage"></a>
+##Git Usage
+
+An overview of how to use Git to participate in Shiro development. Beginners, do't be afraid - you cannot accidentally destroy the actual code repository, because you are working with a local copy as an anonymous user. Therefore, you do not have the system permissions to change anything. You can only update your local repository and compare your revisions with the real repository. The [Download Shiro](download.html "Download") page explains how to check-out the code base and build your local copy.
+
 <a name="HowtoContribute-gettingStartedGit"></a>
-###Getting Started
+###Git, Getting Started
```

Comment:
space after `#`, maybe "getting started with git".
If this is "getting started with git", where is "getting started with code/development/etc"?